subprojects/packagefiles/x-plane-sdk/meson_options.txt

```diff
@@ -0,0 +1,4 @@
+option('xplm-level', type: 'integer', min: 0, max: 400, value: 0,
+      description: 'The XPLM compatibility level to use. See the documentation.')
+option('cpp', type: 'feature', value: 'disabled',
```

Comment:
This started working in 0.60.0: https://github.com/mesonbuild/meson/commit/8a0d12ec295780977d0e2cb7e2f64f1e3a3e0067

Before then it said "Option name cpp is reserved."

It is not clear to me what the commit tried to do or how it tried to do it. There is no FeatureNew for this, nor release notes documentation, and the functionality is not tested, so it would appear to be an accidental side effect...

@xclaesse thoughts?

Reply:
Huh. I will admit, it didn't occur to me to test with a version so old, but it makes sense. I wonder if any distros even have a version pre-0.60.0 packaged still?

Reply:
Debian oldstale (not a typo) has 0.56, Ubuntu 20.04 LTS (Long Term Suckage) has 0.53.

Some people care, some do not. This is not exactly core infrastructure here so I doubt anyone will blink twice if, unlike zlib, this package doesn't build with prehistoric mesons. Even if the distros exist, it doesn't follow that x-plane-sdk users use those distros.

Reply:
From experience: some people do some very cursed stuff in building X-Plane plugins. libacfutils, AFAIK, just does not build if your compilers and stuff are too new. and it has tarballs and stuff in tree. wild stuff.

but yeah i'll just make it `0.60.0` and call it a day

Reply:
the kinds of people crazy enough to build X-Plane plugins with super old distros are probably not the same kinds of people to use meson